### PR TITLE
[RIC] 1.4.3 Related Product List

### PR DESCRIPTION
**Pull request by bishalkg**
[RIC] 1.4.3 Related Product List

### CODE REVIEW

File: client/dist/styles.css [hunk 1]
```diff
@@ -442,37 +451,78 @@ body {
...
+
+.star-for-button {
+  z-index: 3;
+  opacity: 10;
+  font-size: 2.0rem;
+  color: green;
+  transition: font-size 0.8s;
+}
+
+@keyframes spin {
```
Comment (ya-liu) [inline client/dist/styles.css:478]: spinning stars rule

File: client/src/components/RelatedItemsAndOutfit/ComparisonModal.jsx [hunk 2]
```diff
@@ -1,9 +1,185 @@
...
+  // const shouldDisplayModalData = Object.keys(modalRelProd).length !== 0;
+  useEffect(() => {
+    if (Object.keys(modalRelProd).length !== 0) {
+      const currProdKey = Object.keys(currProd)[0];
+      const relProdKey = Object.keys(modalRelProd)[0];
+      const currEntries = currProd[currProdKey];
+      const relEntries = modalRelProd[relProdKey];
+      const currProdChars = Object.keys(currProd[currProdKey]);
+      const relProdChars = Object.keys(modalRelProd[relProdKey]);
+      const allChars = Array.from(new Set([...currProdChars, ...relProdChars]));
```
Comment (ya-liu) [inline client/src/components/RelatedItemsAndOutfit/ComparisonModal.jsx:19]: Nice combo of ES6 syntax

File: client/src/components/RelatedItemsAndOutfit/UserOutfits/UserOutfit.jsx [hunk 3]
```diff
@@ -1,43 +1,50 @@
...
-)}
+        {currOutfits.map((outfit) => (
+          <ListCard
+            key={outfit[0].id}
+            prodInfo={outfit[0]}
+            prodStyles={outfit[1]}
+            prodMeta={outfit[2]}
+          >
+            <button onClick={(e) => removeOutfit(outfit[0].id)} type="button" className="card-button">X</button>
+          </ListCard>
```
Comment (ya-liu) [inline client/src/components/RelatedItemsAndOutfit/UserOutfits/UserOutfit.jsx:39]: JSX syntax?

**Review by ya-liu — APPROVED** [review]

File: client/src/components/RelatedItemsAndOutfit/UserOutfits/UserOutfit.jsx (hunk 3 above)
Comment (bishalkg) [reply client/src/components/RelatedItemsAndOutfit/UserOutfits/UserOutfit.jsx:39]: yup, button type button XD

Comment (bishalkg) [reply client/src/components/RelatedItemsAndOutfit/UserOutfits/UserOutfit.jsx:39]: or wait did i miss something?

**Review by dho1994 — APPROVED** [review]

**Pull request merged**